src/ns/index.ts

```diff
@@ -13,7 +13,7 @@ export class Namespace<GP, TP, SP, OP> {
     protected ableton: Ableton,
     protected ns: string,
     protected nsid?: string,
-  ) {}
+  ) { }
```

Comment:
There seem to be some formatting changes in files that you haven't touched. Could you try running Prettier to reformat them?